code/modules/mob/living/carbon/xenomorph/death.dm

```diff
@@ -137,25 +137,31 @@
 
 /mob/living/carbon/xenomorph/gib(datum/cause_data/cause = create_cause_data("gibbing", src))
 	var/obj/effect/decal/remains/xeno/remains = new(get_turf(src))
-	remains.icon = icon
+	var/icon_path = 'icons/mob/xenos/xenomorph_64x64.dmi'
```

Comment:
```suggestion
	var/icon_path
```
The `mob_size` check will always override this so no point specifying it to suggest changing it would ever do anything.